lib/mongodb.js

```diff
@@ -538,7 +545,7 @@ class MongoDB {
 	/**
 	 * Multi remove items from the database
 	 * @param {Model} model Model instance
-	 * @param {filter} filter mongodb filter
+	 * @param {{filter: {value: {in: string[]}}}} filter mongodb filter
```

Comment:
```suggestion
	 * @param {Object} filter mongodb filter
```
El tipo de dato es un `Object` si `Filter` no es una clase no puede ser el tipo del parámetro.